src/ansys/geometry/core/connection/launcher.py

```diff
@@ -380,10 +380,11 @@ def launch_modeler_with_discovery(
         the latest. Default is ``None``.
         If a specific product version is requested but not installed locally,
         a SystemError will be raised.
-        Possible values:
-        Ansys products Versions and their corresponding int values:
-        23.2 -> value : 232
-        24.1 -> value : 241
+
+        **Ansys products versions and their corresponding int values:**
+
+        * 23.2 -> value : ``232``
+        * 24.1 -> value : ``241``
```

Comment:
It's kind of obvious that it is the int equivalent right? Also, to be uniform with the other lists of equivalences I would put first the int value and then what it represents.
```suggestion
        * ``232`` : 23.2
        * ``241`` : 24.1
```